static/app/views/issueList/utils/useFetchIssueTags.tsx

```diff
@@ -0,0 +1,303 @@
+    ...Object.values(currentTags).map(tag => tag.key),
+    ...Object.values(SEMVER_TAGS).map(tag => tag.key),
+  ].sort();
+  const builtInTags = getBuiltInTags(org);
+
+  const tagCollection: TagCollection = {
+    [FieldKey.IS]: {
+      ...builtInTags[FieldKey.IS],
+      key: FieldKey.IS,
+      name: 'Status',
+      values: Object.values(IsFieldValues),
+      maxSuggestedValues: Object.values(IsFieldValues).length,
+      predefined: true,
+    },
+    [FieldKey.HAS]: {
+      ...builtInTags[FieldKey.HAS],
+      key: FieldKey.HAS,
+      name: 'Has Tag',
+      values: hasFieldValues,
+      predefined: true,
+    },
+    [FieldKey.ASSIGNED]: {
+      ...builtInTags[FieldKey.ASSIGNED],
+      key: FieldKey.ASSIGNED,
+      name: 'Assigned To',
+      values: assigneeFieldValues,
+      predefined: true,
+    },
+    [FieldKey.ASSIGNED_OR_SUGGESTED]: {
+      ...builtInTags[FieldKey.ASSIGNED_OR_SUGGESTED],
+      name: 'Assigned or Suggested',
+      isInput: true,
+      values: assigneeFieldValues,
+      predefined: true,
+    },
+    [FieldKey.BOOKMARKS]: {
+      ...builtInTags[FieldKey.BOOKMARKS],
+      name: 'Bookmarked By',
+      values: bookmarksValues,
+      predefined: true,
+    },
+    [FieldKey.ISSUE_CATEGORY]: {
+      ...builtInTags[FieldKey.ISSUE_CATEGORY],
+      name: 'Issue Category',
+      values: [
+        IssueCategory.ERROR,
+        IssueCategory.PERFORMANCE,
+        IssueCategory.REPLAY,
+        IssueCategory.CRON,
+      ],
+      predefined: true,
+    },
+    [FieldKey.ISSUE_TYPE]: {
+      ...builtInTags[FieldKey.ISSUE_TYPE],
+      name: 'Issue Type',
+      values: [
+        IssueType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES,
+        IssueType.PERFORMANCE_N_PLUS_ONE_API_CALLS,
+        IssueType.PERFORMANCE_CONSECUTIVE_DB_QUERIES,
+        IssueType.PERFORMANCE_SLOW_DB_QUERY,
+        IssueType.PERFORMANCE_RENDER_BLOCKING_ASSET,
+        IssueType.PERFORMANCE_UNCOMPRESSED_ASSET,
+        IssueType.PERFORMANCE_ENDPOINT_REGRESSION,
+        IssueType.PROFILE_FILE_IO_MAIN_THREAD,
+        IssueType.PROFILE_IMAGE_DECODE_MAIN_THREAD,
+        IssueType.PROFILE_JSON_DECODE_MAIN_THREAD,
+        IssueType.PROFILE_REGEX_MAIN_THREAD,
+        IssueType.PROFILE_FUNCTION_REGRESSION,
+      ].map(value => ({
+        icon: null,
+        title: value,
+        name: value,
+        documentation: getIssueTitleFromType(value),
+        value,
+        type: ItemType.TAG_VALUE,
+        children: [],
+      })) as SearchGroup[],
+      predefined: true,
+    },
+    [FieldKey.LAST_SEEN]: {
+      ...builtInTags[FieldKey.LAST_SEEN],
+      name: 'Last Seen',
+      values: [],
+      predefined: false,
+    },
+    [FieldKey.FIRST_SEEN]: {
+      ...builtInTags[FieldKey.FIRST_SEEN],
+      name: 'First Seen',
+      values: [],
+      predefined: false,
+    },
+    [FieldKey.FIRST_RELEASE]: {
+      ...builtInTags[FieldKey.FIRST_RELEASE],
+      name: 'First Release',
+      values: ['latest'],
```

Comment:
This is definitely out of the scope of this PR, but we should really consider if some of these predefined values are outdated. `"latest"` should not be the _only_ option for releases